2 - ListsLogicAndIterations/Code/LogicAndInterations.cs

```diff
@@ -1,36 +1,96 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace ListsLogicAndIterations.Code
 {
 	public static class LogicAndInterations
 	{
 		public static int For_SumTwoListsAndAllValueThatAreNotFour(int[] numberArray, List<int> numberList)
 		{
-			throw new NotImplementedException();
+            int result = 0;
+            foreach(int i in numberArray)
+            {
+                if (i != 4)
+                    result += i;
+            }
+            foreach(int i in numberList)
+            {
+                if (i != 4)
+                    result += i;
+            }
+            return result;
```

Comment:
Though what you did works, what I was hoping to see was a `for` statement, not a `foreach`. 

https://www.dotnetperls.com/for